pysrc/models/metanome_run.py

```diff
@@ -98,6 +108,30 @@ def __iter__(self) -> Iterator[MetanomeRun]:
     @property
     def baseline(self) -> MetanomeRun:
         return next(run for run in self.runs if run.configuration.is_baseline)
+    
+    def tuples_to_remove(self) -> dict[MetanomeRun, tuple[float, float, float, float]]:
+        """Returns the average number (absolute & relative) of rows (total & unique) to remove such that false positive INDs become real
+        The form of a dict entry is (absolute_total, relative_total, absolute_distinct, relative_distinct)"""
+        baseline = self.baseline
+        results: dict[MetanomeRun, tuple[float, float, float, float]] = {}
+        for run in self.runs:
+            tuples_to_remove.tuples_to_remove(baseline_config=baseline.configuration, experiment=run)
+            run_results: dict[IND, list[ErrorMetric]] = {ind: ind.errors for ind in run.results.inds}
+            tuples_to_remove_errors: list[TuplesToRemove] = [
+                error
+                for sublist
+                in run_results.values()
+                for error
+                in sublist
+                if isinstance(error, TuplesToRemove)
+            ]
+            # Consider whether a mean is appropriate here (also consider sum or other metrics)
```

Comment:
Ideally, we would like to know this not on a per-IND level but over all INDs.
However as we've discussed this would be rather costly.
Would be nice to keep this in consideration for the future....